mapr/README.md

```diff
@@ -0,0 +1,97 @@
+
+2. [Restart the Agent][4].
+
+#### Log collection
+
+MapR uses fluentD for logs. Use the [fluent datadog plugin][11] to collect MapR logs.
+The following command downloads and installs the plugin into the right directory.
+
+`curl https://raw.githubusercontent.com/DataDog/fluent-plugin-datadog/master/lib/fluent/plugin/out_datadog.rb -o /opt/mapr/fluentd/fluentd-<VERSION>/lib/fluentd-<VERSION>-linux-x86_64/lib/app/lib/fluent/plugin/out_datadog.rb`
+
+Then update the `/opt/mapr/fluentd/fluentd-<VERSION>/etc/fluentd/fluentd.conf` with the following section.
+
+```
+<match *>
+  @type copy
+  <store> # This section is here by default and sends the logs to ElasticCache for Kibana.
+    @include /opt/mapr/fluentd/fluentd-1.4.0/etc/fluentd/es_config.conf
+    include_tag_key true
+    tag_key service_name
+  </store>
+  <store> # This new section also forwards the logs to Datadog
+    @type datadog
+    @id dd_agent
+    include_tag_key true
+    dd_source mapr
+    dd_tags "flo:test"
+    service <YOUR_SERVICE_NAME>
+    api_key <YOUR_API_KEY>
+  </store>
+```
+
+Refer to [fluent_datadog_plugin][11] documentation for more details about the option you can use.
+
+
+### Validation
+
+[Run the Agent's status subcommand][5] and look for `mapr` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][13] for a list of default metrics provided by this integration.
+
+### Service Checks
+
+- `mapr.can_connect`:
+Returns CRITICAL if the Agent fails to connect to the MapR monitoring streams, otherwise returns UP.
+
+### Events
+
+The MapR check does not include any events.
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][6].
+
+[1]: https://mapr.com
+[2]: https://app.datadoghq.com/account/settings#agent
+[3]: https://github.com/DataDog/integrations-core/blob/master/mapr/datadog_checks/mapr/data/conf.yaml.example
+[4]: https://docs.datadoghq.com/agent/guide/agent-commands/?tab=agentv6#start-stop-and-restart-the-agent
+[5]: https://docs.datadoghq.com/agent/guide/agent-commands/?tab=agentv6#agent-status-and-information
+[6]: https://docs.datadoghq.com/help
+[7]: https://mapr.com/docs/61/MapR_Streams/MapRStreamsPythonExample.html
+[8]: https://mapr.com/docs/61/SecurityGuide/GeneratingServiceTicket.html
+[9]: https://mapr.com/docs/60/MapR_Streams/MapRStreamCAPISetup.html
+[10]: https://mapr.com/docs/61/AdministratorGuide/c-managing-users-and-groups.html
+[11]: https://www.rubydoc.info/gems/fluent-plugin-datadog
+[12]: https://mapr.com/docs/61/AdvancedInstallation/SettingUptheClient-install-mapr-client.html
+[13]: https://github.com/DataDog/integrations-core/blob/master/mapr/metadata.csv
+[14]: http://upstart.ubuntu.com/cookbook/#environment-variables
+[15]: https://www.freedesktop.org/software/systemd/man/systemd.service.html#Command%20lines
```

Comment:
Since this integration does have integration/e2e tests. Do you mind adding a dev readme in test folder about how to setup an local environment for testing MapR ?

Reply:
👍